app/Helpers/Number.php

```diff
@@ -60,6 +60,16 @@ public static function bitsToHuman(int|float $bits, int $precision = 0, ?int $ma
         return sprintf('%s %s', static::format($bits, $precision, $maxPrecision), $units[$i]);
     }
 
+    /**
```

Comment:
Instead of rolling our own helper method you can use the `Number::fileSize()` helper provided by Laravel https://laravel.com/docs/12.x/helpers#method-number-file-size so this method can be removed.